arch/arm/src/kinetis/kinetis_edma.c

```diff
@@ -0,0 +1,1464 @@
+/****************************************************************************
+ * arch/arm/src/kinetis/kinetis_edma.c
+ *
+ *   Copyright (C) 2019, 2021 Gregory Nutt. All rights reserved.
+ *   Authors: Gregory Nutt <gnutt@nuttx.org>
+ *            David Sidrane <david.sidrane@nscdg.com>
+ *
```

Comment:
Please include the Apache License here and keep the Freescale/NXP BSD License below. This is how Samsung did when they released TIzenRT as Apache using NuttX BSD License.

Reply:
"